terraform/locals.tf

```diff
@@ -3,21 +3,6 @@
 
 locals {
 
-  admins = {
-    for user in var.admins : user => "admin"
-  }
-
-  branch_protections = {
```

Comment:
it looks like we’re removing branch protection entirely. Is that expected?

Reply:
it was never really used directly, as can be seen in the `tf plan` comment.
However, I fixed the `plan` gha to run on changes to `.tf` files and also removed the `enable_branch_protection` from every mention.